src/main/java/core/basesyntax/interfaces/Drawable.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.interfaces;
+
+public interface Drawable {
+    String WORD_SEPARATOR = " ";
+    String COMMA = ",";
```

Comment:
generally, it's bad practice to use interface constants. Here are some details: https://stackoverflow.com/questions/2659593/what-is-the-use-of-interface-constants